kokoro/macos_external/kokoro_build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+
+# Display commands being run.
+# WARNING: please only enable 'set -x' if necessary for debugging, and be very
+#  careful if you handle credentials (e.g. from Keystore) with 'set -x':
+#  statements like "export VAR=$(cat /tmp/keystore/credentials)" will result in
+#  the credentials being printed in build logs.
+#  Additionally, recursive invocation with credentials as command-line
+#  parameters, will print the full command, with credentials, in the build logs.
+# set -x
+
+
+# Code under repo is checked out to ${KOKORO_ARTIFACTS_DIR}/github.
+# The final directory name in this path is determined by the scm name specified
+# in the job configuration.
+cd ${KOKORO_ARTIFACTS_DIR}/github/flutter-intellij-kokoro
```

Comment:
Just clarifying - from reading the above, the job name is `flutter-intellij-kokoro`?

Reply:
Yes. It is the directory into which this repo is checked out when Kokoro runs.